app/views/partner_users/invitations/edit.html.erb

```diff
@@ -0,0 +1,50 @@
+<div class='login-page login-page--partner'>
+  <div class="login-box">
+    <div class="login-logo">
+      <img id="MastheadLogo" src="/img/partnerbase-logo.svg" style="width: 80%; height: 80%; " alt="Partner" title="Partner" class="serv_icon"/>
+    </div>
+
+    <div class="login-box-body">
+      <div class="form-group has-feedback">
+        <%= render partial: "shared/flash" %>
+
+        <%= simple_form_for resource, as: resource_name, url: invitation_path(resource_name), html: { method: :put } do |f| %>
+
+          <div class="login-box">
+            <!-- /.login-logo -->
```

Comment:
Do we want to remove this?

Reply:
@scooter-dangle yep :). Actually I'll probably need to fix the devise views, I noticed that https://diaper.app/account_requests/new styling off.